ServerSide/yoty/WebApi/Controllers/BidsController.cs

```diff
@@ -176,6 +176,29 @@ public async Task<ActionResult> DeleteSupplierProposal(string bidId, string prop
             return this.StatusCode(StatusCodes.Status405MethodNotAllowed, response.SuccessOrFailureMessage);
         }
 
+        [HttpPost]
+        [Route("{bidId}/vote")]
+        public async Task<ActionResult> VoteForSupplier(VotingRequest votingRequest)
+        {
+            if (votingRequest.BidId == null || votingRequest.BuyerId == null)
```

Comment:
Add a TODO / Add to this PR:
limit participant to only vote for a single proposal